backend/controllers/SiteController.php

```diff
@@ -1,7 +1,7 @@
 <?php
 namespace backend\controllers;
 
-use Yii;
+use yii;
```

Comment:
This change is not correct, it is to include the class `Yii` not the namespace `yii`.
Same applies to all other changes below.